packages/ui/src/components/vuestic-components/va-input/VaInput.demo.vue

```diff
@@ -256,6 +256,7 @@
         :messages="messages"
       />
     </VbCard>
+<!--    TODO: Figure out why demo overrides user-select: none-->
```

Comment:
What do you mean here?

Reply:
Disabled VaInput text is selectable in our demo even though user-select correctly set to none.

Disabled VaInput works as intended everywhere else.

Looks like our VbPage overrides `user-select` property.

It was a side note to figure it out,

Thanks I will remove it.